src/go/MONIT/alert.go

```diff
@@ -545,21 +572,34 @@ func parseConfig(verbose int) {
 	configJSON.SeverityLevels["info"] = 0
 	configJSON.SeverityLevels["warning"] = 1
 	configJSON.SeverityLevels["medium"] = 2
+	configJSON.SeverityLevels["high"] = 3
+	configJSON.SeverityLevels["critical"] = 4
+	configJSON.SeverityLevels["urgent"] = 5
 	configJSON.httpTimeout = 3 // 3 seconds timeout for http
 
-	if configFile != "" {
-		jsonFile, e := os.Open(configFile)
-		if e != nil {
-			log.Fatalf("Config File not found, error: %s", e)
+	if *generateConfig {
+		config, err := json.Marshal(configJSON)
```

Comment:
Since you'll write config to the file it is better to write it with proper identation for better user readability, e.g. `json.MarshalIndent(configJSON, "", "    ") `

Reply:
Fixed